modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/messages/StatisticsColumnData.java

```diff
@@ -243,15 +243,15 @@ public long createdAt() {
 
         switch (reader.state()) {
             case 0:
-                cardinality = reader.readInt("cardinality");
+                createdAt = reader.readLong("createdAt");
```

Comment:
I think it will not work in case we try read old message. Need to write RU tests

Reply:
We won't even try to read it cause statistics version was incremented in IgniteStatisticsPersistenceStoreImpl. So all statistics data will be removed from local metastore without unmarshalling and collected again. We can even swith from nevest version to oldest one in that part.